modules/quickstart/pages/dfx-deploy-quickstart.adoc

```diff
@@ -0,0 +1,389 @@
+= Introducing a simplified developer workflow
+:experimental:
+// Define unicode for Apple Command key.
+:commandkey: &#8984;
+:proglang: Motoko
+:platform: Internet Computer platform
+:IC: Internet Computer
+:company-id: DFINITY
+:sdk-short-name: SDK Developer Workflow Preview
+:sdk-long-name: DFINITY Canister Software Development Kit (SDK)
+ifdef::env-github,env-browser[:outfilesuffix:.adoc]
+
+[[preview-intro]]
+The {sdk-short-name} release includes a preview of a simplified developer workflow.
+As part of our developer community, we encourage you to try out this new workflow and provide us with feedback on your experience. 
+
+The new developer workflow is intended to streamline the development process and make it easier for you to iterate quickly on your application design and implementation.
+
+== Before you begin
+
+The {sdk-short-name} release is built from the same code base as the publicly-available release of the {sdk-long-name}.
+However, this version of the software includes features and changes that are not yet supported and might be known to be incomplete or unstable.
+In addition, the {sdk-short-name} release is not intended to work with any existing projects you might have previously compiled or deployed.
+
+If possible, you should install the {sdk-short-name} preview release in a completely clean environment with no other versions of the {sdk-long-name} installed.
+For information about removing the current software before installing a new release, see link:../developers-guide/install-upgrade-remove{outfilesuffix}#remove[Removing the software] before continuing to <<Download and install>> the {sdk-short-name} release.
+
+Before you download and install the {sdk-short-name}, verify the following:
+
+* You have an internet connection and access to a shell terminal on your local macOS or Linux computer.
+
+* You have `+node.js+` installed if you want to include the default template files for front-end development in your project.
+
+* You have stopped any {IC} network processes running on the local
+computer.
+
+== Getting started
+
+To provide feedback on the simplified developer workflow, you must have the _not publicly promoted_ version of the {sdk-short-name} available on your local computer.
+
+[[download-and-install]]
+=== Download and install
+
+To download the {sdk-short-name}, you must specify the version using an environment variable. 
+
+To download and install:
+
+[arabic]
+. Open a terminal shell on your local computer, if you don’t already have one open.
++
+For example, open Applications, Utilities, then double-click *Terminal* or press kbd:[{commandkey} + spacebar] to open Search, then type `terminal`.
+. Download and install the {sdk-short-name} by running the following command:
++
+[source,bash]
+----
+DFX_VERSION=0.6.8-preview sh -ci "$(curl -fsSL https://sdk.dfinity.org/install.sh)"
```

Comment:
Are we committing to having a `-preview` version? There's not yet any differences I think (deploy is stable).

Reply:
I thought we had agreed to that approach. If it goes into the release tomorrow, the tutorials will not be updated. until the next release at the earliest.
From what I remember, though, we were going to release the feature with minimal documentation (this standalone thing) and hopefully give the community a chance to provide feedback (in case we wanted to tweak the implementation further before making a mass change to documentation--for example, see https://github.com/dfinity-lab/sdk/issues/1015).